src/ints/mouse.cpp

```diff
@@ -1099,6 +1190,135 @@ Bitu UIR_Handler(void) {
 	return CBRET_NONE;
 }
 
+// ***************************************************************************
+// External notifications
+// ***************************************************************************
+
+void Mouse_SetSensitivity(int32_t sensitivity_x, int32_t sensitivity_y) {
+    static constexpr float MIN = 0.01f;
+    static constexpr float MAX = 100.0f;
```

Comment:
`min` and `max` should be lowercase, and isn't `static` a bit of an overkill here?